automatic/filezilla/tools/chocolateyUninstall.ps1

```diff
@@ -1,7 +1,7 @@
 ﻿$ErrorActionPreference = 'Stop'
 
 $packageName         = 'filezilla'
-$softwareNamePattern = 'FileZilla Client*'
+$softwareNamePattern = 'FileZilla 3*'
```

Comment:
This will probably break again when the software will update to v4 or higher

Reply:
Agreed. @AdmiringWorm mentioned as part of the PR review that this is likely something that we want to bring out as part of the update.ps1 script, but that this could be achieved in a separate scope of work.